manifests/f/FrostWire/FrostWire/6.12.0/FrostWire.FrostWire.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: FrostWire.FrostWire
+PackageVersion: 6.12.0

```

```suggestion
PackageVersion: 6.12.0.318
```
This must match Control Panel.